frontend/src/features/contacts/hooks/useContactMethodHelpers.ts

```diff
@@ -0,0 +1,47 @@
+import * as API from 'constants/API';
+import {
+  ContactMethodTypes,
+  emailContactMethods,
+  phoneContactMethods,
+} from 'constants/contactMethodType';
+import useLookupCodeHelpers from 'hooks/useLookupCodeHelpers';
+import { useMemo } from 'react';
+import { ILookupCode } from 'store/slices/lookupCodes';
+
+/**
+ * Extending default ILookupCode interface to enforce the presence of `displayOrder` property
+ * This is okay here because both code tables have this field (not the case with other code tables)
+ */
+interface IHasDisplayOrder extends Omit<ILookupCode, 'displayOrder'> {
```

Comment:
Is there a reason not to do this on the server side?

Reply:
I guess I could have modified the server to always have displayOrder but I believe not all lookup codes have order so I went for the least disruptive way of modelling it

Reply:
Sorry, I meant why not have the server return the list sorted by display order, then you would not need to return the display order at all.

Reply:
I see. I didn't see any existing example where we do that. Also, another problem I noticed is that most lookup codes we do not check for **isDisabled** so there's potential to return old/stale lookup codes. I added the check on the ones I created.

I can sort by display order in the server in my next PR for the lookup codes I need. The others would need a separate ticket, no?

Reply:
Checked and I already sort by display order in the server (for the ones I use in my PR)

```cs

  public IEnumerable<PimsCountry> GetCountries()
        {
            return this.Context.PimsCountries.AsNoTracking().OrderBy(a => a.DisplayOrder).ThenBy(a => a.CountryCode).ToArray();
        }

        /// <summary>
        /// Get all provinces sorted by DisplayOrder and Code
        /// </summary>
        public IEnumerable<PimsProvinceState> GetProvinces()
        {
            return this.Context.PimsProvinceStates.AsNoTracking().Where(c => c.IsDisabled != true).OrderBy(a => a.DisplayOrder).ThenBy(a => a.ProvinceStateCode).ToArray();
```

Reply:
NOTE - PimsCountry doesn't have **isDisabled** (which it should I think) so I couldn't include that in the country query

Reply:
yeah, don't need to fix it now, just wanted to make sure you are aware so you can save yourself some time/effort/loc on future stories.